### PR TITLE
Migrate/zh/cennznet executor

### PR DESCRIPTION
**Pull request by mzxyz**
*  Update crate names and compile passed
*  Ignore 2 test cases, other test cases pass
    - full_wasm_block_import_works
    - full_native_block_import_works

```
---- tests::full_native_block_import_works stdout ----
thread 'tests::full_native_block_import_works' panicked at 'assertion failed: `(left == right)`
  left: `4197489990000000`,
 right: `4197489980001000`', executor/src/lib.rs:522:4
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

These errors will be fixed in another PR

### CODE REVIEW

**Review by cowboy-bebug — APPROVED** [review]
LGTM! The values differ by `9_999_000`, which we need to spend more time to investigate

**Comment by mzxyz** [comment]
@royyang-centrality  once this one merged, you can rebase the code and test `cennznet-cli`

File: executor/Cargo.toml [hunk 1]
```diff
@@ -5,31 +5,37 @@ authors = ["Centrality Developers <support@centrality.ai>"]
...
-primitives = { package = "sp-core", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
-trie = { package = "sp-trie", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
+trie-root = "0.15.2"
+
+# internal dependencies
 cennznet-primitives = { path = "../primitives" }
 cennznet-runtime = { path = "../runtime" }
 
+#frame dependencies
+# runtime-io = { package = "sp-io", git = "https://github.com/plugblockchain/plug-blockchain", branch = "1.0.0-rc2" }
```
Comment (jordy25519) [inline executor/Cargo.toml]: Why some deps are commented?

**Review by jordy25519 — APPROVED** [review]
Changes look sensible. Wondering why some dependencies are commented out though?

File: executor/Cargo.toml (hunk 1 above)
Comment (mzxyz) [reply executor/Cargo.toml]: Oh, we forgot it, these should be removed, done

**Pull request merged**